src/helm/benchmark/static/schema_speech.yaml

```diff
@@ -320,6 +320,28 @@ run_groups:
       when: "2019"
       language: English
 
+  - name: voxceleb2
+    display_name: VoxCeleb2
+    description: >
+      AudioCaps is a large-scale dataset of about 46K audio clips to human-written text pairs collected 
+      via crowdsourcing on the AudioSet dataset, which covers a wide range of human and animal sounds, 
+      musical instruments and genres, and common everyday environmental sounds. 
+      ([Kim et al, 2019](https://aclanthology.org/N19-1011.pdf)).
+      VoxCeleb is an audio-visual dataset consisting of short clips of human speech, extracted from 
+      interview videos uploaded to YouTube.([Chung et al, 2018](https://www.robots.ox.ac.uk/~vgg/publications/2018/Chung18a/chung18a.pdf))
+    metric_groups:
+      - accuracy
+      - general_information
+    environment:
+      main_name: cider
```

Comment:
I think you want `quasi_exact_match` as your `main_name` (main metric). `exact_match` doesn't work because models can output both `true` and `True`, and `exact_match` is case sensitive. Not sure what `cider` is.

Reply:
thanks for the review, yifan! I've changed this task to mcqa